Atomic/diff.py

```diff
@@ -2,24 +2,63 @@
 import sys
 import rpm
 from filecmp import dircmp
-
 from . import util
 from . import mount
 from . import Atomic
 
+''' This file will handle both dbus calls and regular calls from Atomic '''
+
```

Comment:
No reason for this, it has no idea that dbus is calling it.  Just remove comment.